pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotAdministrator.java

```diff
@@ -185,39 +173,23 @@ private void printVersion() {
     }
   }
 
+  public void printUsage() {
+    LOGGER.info("Usage: pinot-admin.sh <subCommand>");
+    LOGGER.info("Valid subCommands are:");
```

Comment:
The sub command info seems missing here?

Reply:
+1 good catch

Reply:
good catch. let me fix this

Reply:
this should be fixed